### DISCUSSION TITLE
Rest API [Aug 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3125] [Discord Threads]
> 🙋‍♂️ **ghost**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@361928900588077057> what was the issue? help us with details here, we will look into.   
####    
#### https://github.com/nocodb/nocodb/issues  
#  
> 🙋‍♂️ **ghost**  
#### It would return an error when I tried to generate the API. I’ll test again in the morning.  
#  
> 💁 **o1lab**  
#### logs from your server side will help  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1004575154187280475)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
